ccswbs/hjckrrh
# Iss767 update fontawesome to v3.7 module and v5.2 library

# Changes
- updated the makefile to use the latest version of the module(v3.7) and library(5.2)
- updated S1 (Share this Page) to use .fab class instead of .fa
- updated S3, S4, and S6 to use .fab class except for rss which uses .fa class (If adding a new icon to the Social Media content type which does not use .fab, remember to add the icon name to the ```$fa_icons``` array in the ```process_icon_bundle()``` function in the template.php file)
- added UG Font Awesome feature with default configuration for Font Awesome (uses Web Fonts with CSS)

# Release Instructions
- For new sites, UG Font Awesome will be enabled by default.
- For existing sites, there is an update_hook which enables the feature.
- If you cannot enable UG Font Awesome on a site, you can manually apply the change by navigating to /admin/config/content/fontawesome and ensuring "Web Fonts with CSS" is selected

# Testing
- check that the old icons still work, and that new ones do as well for Share This Page, Follow Us (both S3 and S4 versions), and Directory (S6)
- check that Font Awesome icons in the footer are not broken
- check that Font Awesome configuration has "Web Fonts with CSS" selected
